utils/chaos_recommender/chaos_recommender.py

```diff
@@ -54,16 +60,47 @@ def read_configuration(config_file_path):
     prometheus_endpoint = config.get("prometheus_endpoint", "")
     auth_token = config.get("auth_token", "")
     scrape_duration = config.get("scrape_duration", "10m")
-    chaos_tests = config.get("chaos_tests" , {})
+    chaos_tests = config.get("chaos_tests", {})
+    json_output = config.get("JSON_output", False)
     return (namespace, kubeconfig, prometheus_endpoint, auth_token, scrape_duration,
-            chaos_tests, log_level)
+            chaos_tests, log_level, json_output)
+
 
 def prompt_input(prompt, default_value):
     user_input = input(f"{prompt} [{default_value}]: ")
     if user_input.strip():
         return user_input
     return default_value
 
+
+def make_json_output(file, namespace, kubeconfig, prometheus_endpoint, chaos_tests, scrape_duration, queries, analysis_data):
+    data = {
+        "inputs": [
+            {
+                "namespace": namespace,
+                "kubeconfig": kubeconfig,
+                "prometheus_endpoint": prometheus_endpoint,
+                "scrape_duration": scrape_duration,
+                "chaos_tests": chaos_tests
+            }
+        ],
+        "analysis_start": [
+            {
+                "cpu_query": queries[0],
+                "cpu_limit_query": queries[1],
+                "memory_query": queries[2],
+                "memory_limit_query": queries[3]
+            }
+        ],
+        "profiling": analysis_data[0],
+        "heatmap_analysis": analysis_data[1],
+        "recommendations": analysis_data[2]
+    }
+    file = f"utils/chaos_recommender/recommender_output/{file}"
```

Comment:
@jtydlack We might want to append a timestamp to the file name to allow storing data from multiple runs. Thoughts?

Reply:
There is already a timestamp but it is created on different spot so I will move it to this function so it is all at one place